.circleci/config.yml

```diff
@@ -97,6 +97,9 @@ jobs:
       - run:
           name: Install Prereqs
           command: sudo apt-get update && sudo apt-get install -y libudev-dev libusb-1.0-0-dev yarn rsync
+      - run:
+          name: Ignore untracked files
+          command: git add . && git commit -m "ci-ignore-untracked-files"
```

Comment:
I don't want these untracked files to be published though. Thoughts on removing them instead?